### ISSUE TITLE
Option to add mystery box to offline player from console

### ISSUE THREAD

**Issue by nixacik** [yapzhenyie/GadgetsMenu#817]
**Is your feature request related to a problem? Please describe.**
yes it is related, since I have a vip system on another server I would like to add so that it is possible to add a mystery box to the player from the console if the player is offline
To add the mystery box / s directly to the database.

**Describe the solution you'd like**
I would like to add it would be possible to execute the command /gmysterybox give <player ... even if the player is offline.

**Additional context**
I will be very grateful if you consider and add sooner.

**Comment by OnlyNoobDied** [comment]
This feature has been supported for a long time.

**Comment by nixacik** [comment]
But those players had to be connected to a given server to give them a mystery box when they were offffline. But I have 5 Lobby server + VIP server where there are orders that will be executed when the payment arrives.
Players cannot connect to the VIP server, so it says "Player not found:" when the payment arrives and it has to give the mystery box to the player.

![image](https://user-images.githubusercontent.com/55744915/100542771-6cd90f80-324c-11eb-8b25-2a84b89c0a2a.png)

**Comment by yapzhenyie** [comment]
1. Which GadgetsMenu version you're using?
2. Did that player ever join in the server after you have GadgetsMenu plugin?

**Comment by nixacik** [comment]
1. 4.6.3 Premium
2. Yes, he joined
![image](https://user-images.githubusercontent.com/55744915/100543149-b9bde580-324e-11eb-86b0-4debf824096d.png)
3. yes i use
![image](https://user-images.githubusercontent.com/55744915/100543219-1caf7c80-324f-11eb-9ca9-6d58f65706bc.png)

**Comment by yapzhenyie** [comment]
Can you try give Mystery Box to other player to see if it works?

**Comment by nixacik** [comment]
Yes, i try it

![image](https://user-images.githubusercontent.com/55744915/100546022-47093600-325f-11eb-9226-f71d9c9bfa36.png)

But only works if the player has ever been on that server.

The player you see in the picture and the picture I sent before are in the database.

**Comment by yapzhenyie** [comment]
@nixacik I might need to know your server ip so I can join your server to test this issue.

**Comment by nixacik** [comment]
IP: mc.pvpmania.eu
(Czech and Slovakia PvP server)

**Comment by yapzhenyie** [comment]
@nixacik I'm inside now. (lobby 2)

**Comment by yapzhenyie** [comment]
`online-mode` is different among servers.

**Comment by nixacik** [comment]
![image](https://user-images.githubusercontent.com/55744915/100618414-1cc97e00-331c-11eb-875c-d2a7d4bd0585.png)
VIP system server

once it works and once it doesn't. There is no player on the server and there is online-mode false

**Comment by yapzhenyie** [comment]
DANYELEEQ_ does not join to the server before after you have changed the `online-mode`.

**Comment by nixacik** [comment]
but players do not connect to the VIP server, they only connect to the Lobby. Will it work when I connect to the Lobby and want to execute these commands on the VIP System server?

Sorry to disturb you. As I mentioned, when it goes, I'll buy it right away.

**Comment by yapzhenyie** [comment]
Typically it will works unless `misspelling` or `online-mode` issue.

**Comment by nixacik** [comment]
I tried to connect for another nick (online mode: false on all servers) and it doesn't work when I execute the command on the VIP System server. I'm through my second nick at the Lobby

![image](https://user-images.githubusercontent.com/55744915/100620440-b98d1b00-331e-11eb-9d14-09047bfc5d9e.png)

![image](https://user-images.githubusercontent.com/55744915/100620492-c578dd00-331e-11eb-905a-54b1e324ba53.png)

**Comment by yapzhenyie** [comment]
@nixacik Again, is your spigot.yml file `bungee-mode` setting set to `true`?

**Comment by nixacik** [comment]
Yes

**Comment by yapzhenyie** [comment]
Is VIP System server connected to your bungeecord server? Did you set the mysql info equally within all servers including VIP System server?

**Comment by nixacik** [comment]
yes, it is 100% connected to bungeecord and the database has the same data everywhere

**Comment by yapzhenyie** [comment]
Is this just a case? Have you tried other player name?

**Comment by nixacik** [comment]
I tried another name. For example, it went to my nickname (because I was already on the VIP server) but it didn't work on others. And I tried it. I also tried mystery dust.
![image](https://user-images.githubusercontent.com/55744915/101049253-a29b3280-3583-11eb-9d2d-c6389d34d1d0.png)

**Comment by yapzhenyie** [comment]
So the issue only happen between your main server and VIP server? If you do it in different hub server, does it work?

**Comment by nixacik** [comment]
yes, the problem partly occurs only between the main server and the VIP server? It doesn't have to be just the Lobby. I have 6 Lobby servers. I tried it on multiple servers.
